src/esp/sim/Simulator.h

```diff
@@ -555,6 +555,20 @@ class Simulator {
    */
   void setObjectBBDraw(bool drawBB, const int objectID, const int sceneID = 0);
 
+  /**
+   * @brief Set the @ref esp::scene:SceneNode::semanticId_ for all visual nodes
+   * belonging to an object.
+   *
+   * @param semanticId The desired semantic id for the object.
+   * @param objectID The object ID and key identifying the object in @ref
+   * esp::physics::PhysicsManager::existingObjects_.
+   * @param sceneID !! Not used currently !! Specifies which physical scene of
+   * the object.
+   */
+  void setObjectSemanticId(uint32_t semanticId,
+                           const int objectID,
+                           const int sceneID);
```

Comment:
```suggestion
                           int objectID,
                           int sceneID);
```

Some shitty IDEs insist on adding the unnecessary `const` to the declaration if you have it in the definition, I hope it's not the case of yours :D